demo/registry.yaml

```
@@ -54,7 +57,10 @@ chains:
     chain_id: cosmoshub-4
     prefix: cosmos
     gas_price: 0.026uatom
-    hd_path: m/44'/108'/0'/6'
+    # Default keyring hd_path for the SDK
+    # This is used for example by `gaiad keys` and most CosmJS webapps
+    # Set this is you want to share the relayer account with a CLI account (not recommended)
+    hd_path: m/44'/108'/0'/0/0
```

```suggestion
    hd_path: m/44'/118'/0'/0/0
```
I believe the coin type is [118](https://github.com/cosmos/cosmos-sdk/blob/377360a391e80787d99bd0a9ce962a18bee8dc77/types/address.go#L46)